src/module/data/item/ability.mjs

```diff
@@ -63,6 +63,7 @@ export default class AbilityModel extends BaseItemModel {
     }, initial: "melee", required: true, blank: false });
     schema.target = new fields.SchemaField({
       type: new fields.StringField({ required: true, blank: false, initial: "self" }),
+      custom: new fields.StringField(),
```

Comment:
Personal preference never to have string fields be `undefined` but if you feel strongly about it then eh.
```suggestion
      custom: new fields.StringField({ required: true }),
```